examples/game/breakout.rs

```diff
@@ -54,30 +54,32 @@ fn main() {
                 .with_run_criteria(FixedTimestep::step(TIME_STEP as f64))
                 .with_system(paddle_movement_system)
                 .with_system(ball_collision_system)
-                .with_system(ball_movement_system),
+                .with_system(apply_velocity_system),
         )
         .add_system(scoreboard_system)
         .add_system(bevy::input::system::exit_on_esc_system)
         .run();
 }
 
 #[derive(Component)]
-struct Paddle {
-    speed: f32,
-}
+struct Paddle;
 
 #[derive(Component)]
-struct Ball {
-    velocity: Vec3,
-}
+struct Ball;
 
 #[derive(Component)]
-enum Collider {
-    Solid,
-    Scorable,
-    Paddle,
+struct Velocity {
+    x: f32,
+    y: f32,
 }
```

Comment:
I don't know if it's good Bevy style, but does it make sense to change this to use `Vec2` or `Vec3`?
It would "play" better with `Transform`.

```suggestion
struct Velocity {
    value: Vec2,
}
```